### PR TITLE
PS-2399: catch job transition exceptions and don't log them to events

### PR DESCRIPTION
**Pull request by MiroCillik**

### CODE REVIEW

File: src/Command/RunCommand.php [hunk 1]
```diff
@@ -184,6 +186,22 @@ private function postJobResult(string $jobId, string $status, JobResult $result,
     {
         try {
             $this->queueClient->postJobResult($jobId, $status, $result, $metrics);
+        } catch (StateTerminalException $e) {
+            $this->logger->debug(
```
[the remark this answers is not in the document]
Comment (MiroCillik) [reply src/Command/RunCommand.php:190]: Nad tym noticom som rozmyslal, ale nebude to tiez prepadavat do Storage eventov?

[the remark this answers is not in the document]
Comment (odinuv) [reply src/Command/RunCommand.php:190]: nemělo by https://github.com/keboola/docker-bundle/blob/67ca48398d2b6668e1807772d728342c5e65e70e/src/Service/LoggersService.php#L44

[the remark this answers is not in the document]
Comment (MiroCillik) [reply src/Command/RunCommand.php]: super, diky, rozmyslal som nad tym noticom, ale toto som prehliadol

File: composer.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
         "keboola/billing-api-php-client": "^0.1",
         "keboola/configuration-variables-resolver": "^1.0.0",
         "keboola/dockerbundle": "dev-odin-library-3@dev",
-        "keboola/job-queue-internal-api-php-client": "^7.3",
+        "keboola/job-queue-internal-api-php-client": "dev-miro-ps-2399",
```
Comment (odinuv) [inline composer.json:26]: update before merge

**Review by odinuv — APPROVED** [review]

**Pull request merged**